src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -127,7 +126,6 @@ private boolean exec() {
                 }
 
                 return
-                    maybeLogin() &&
```

Comment:
I'm not sure that the deletion of this call is safe. Probably, a replacement call base on docker-commons is required